.gitignore

```diff
@@ -16,7 +16,12 @@ _build/**
 *.opt
 *.native
 *.log
-/node_modules/
-/lib/
-/immutable.install
-/pkg/META
+node_modules
+immutable.install
+
+# Ignore bsb generated merlin files
+/reUnit/.merlin
```

Comment:
It's a local subfolder. I'm working with the bs-container guys about pulling it into its own external package at some point.